project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,109 @@
+everything to find something to work on.  Triaging issues is important to
+help both the author convey the real problem or request that they have and
+to help developers easily find something to work on.
+
+Remember there are more ways than just code that you can contribute to an
+open source project and the community would really thank you if you spent
+spend time triaging issues.
+
+
+### Important first steps
+
+Before jumping into an issue very far make sure that the author of the issue
+has provide enough information so that you are able to make an educated 
+reccomendation on how this is to be triaged.  Standard information that *must*
+be included in most issues are things such as:
+
+* docker version
+* docker info
+* host distro ( ubuntu, RHEL, fedora )
+
+A bonus is to ask for a way to reproduce the issue.
+
+Additional information can be requested if needed.  If an issue is totally unable
+to be triaged based on the author's provided information, kindly state that
+they will have to provide the above information and clarification on the problem.
+Close the issue letting them know they can re-open once the information is 
+available so that it is able to be properly triaged.  Closing the issue is ok
+because some people do not have time to respond quickly and there is no use
+leaving an issue open that cannot be acted upon.  Feel free to point them to 
+this document so that they understand the issue is not being closed due to
+maintainers and contributors not wanting to help but because they don't have
+the information to provide help in a meaningful way.
+
+
+### Classification
+
+Classifications can come in a few different flavors.  These are used to help
+prioritize and understand what type of issue this is and how it should be 
+resolved. 
+
+Types of classification:
+* improvement - improvements are not bugs or new features but can drastically 
+improve usability.
+* bug - bugs are bugs.  The cause may or may not be known at triage time so
+debugging should be taken account into the time estimate. 
+* regression - regressions are usually easy fixes as hopefully the action
+worked previously and git history can be used to propose a solution.
+* feature - features are new and shinny.  They are things that the project does
+not currently support.
+
+
+### Time estimate
+
+Everyone who has ever written software knows how time estimates work.  Every PM
+has their own special formula for when a developer gives them an estimate the
+multiply it by a factor of 10 or something like that.  Every developer thinks 
+that you cannot rush greatness.  So why should an open source project place
+time estimates on issues?  Because this is not requiring that an issue should
+be completed in this amount of time by a contributor, it is being used as a
+way for the contributor to know how long thing may take if they wanted to work
+on this task.  Maybe someone feels like hacking on docker over the weekend and
```

Comment:
The sentence starting *"Because this is not.. "* has some clause issues that make it a bit hard to read/understand.  

Maybe something like: "The estimate doesn't require the contributor to solve the issue in the specified amount of time, but rather is a way for the contributor to understand a ballpark time commitment required of them if they choose to work on this task."